package.json

```diff
@@ -12,13 +12,14 @@
   "private": true,
   "dependencies": {
     "@bufbuild/connect-web": "^0.6.0",
+    "@bufbuild/protobuf": "^1.0.0",
     "@grpc/grpc-js": "^1.7.3",
+    "@openfeature/web-sdk": "^0.1.0-experimental",
```

Comment:
I'm confused, the web-sdk repo is empty, where is this coming from?

Reply:
Sorry about that, I initially created a repo for it, but ended up instead converting the `js-sdk` to a monorepo because a lot of the code/interfaces were similar.